frontend/src/views/WheresMyGene/components/HeatMap/index.tsx

```diff
@@ -134,6 +134,14 @@ export default function HeatMap({
       useDirtyRect: true,
     });
 
+    xAxisChart.on("click", function (params) {
+      console.log(params);
+    });
+
+    yAxisChart.on("click", function (params) {
+      console.log(params);
+    });
+
```

Comment:
Debug?

Reply:
Oooh yes! Part of the delete genes/cellTypes stuff sneaked in 😆 Removed! Thank you!